samples/benchmarks/coremark/README.rst

```diff
@@ -171,8 +193,22 @@ CONFIG_APP_MODE_FLASH_AND_RUN - Start CoreMark sample automatically after flashi
    Otherwise, it will wait for the button press.
 
 .. note::
-   The :kconfig:option:`CONFIG_APP_MODE_FLASH_AND_RUN` Kconfig option is always enabled for the PPR core.
-   This core does not use buttons.
+   The :kconfig:option:`CONFIG_APP_MODE_FLASH_AND_RUN` Kconfig option is always enabled for the PPR and FLPR cores on the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
+   These cores on the ``nrf54h20dk/nrf54h20/cpuapp`` board target do not use buttons and LEDs.
+
+.. _SB_CONFIG_APP_CPUFLPR_RUN:
+
+SB_CONFIG_APP_CPUFLPR_RUN - Enable execution for the FLPR core
+   Enable the benchmark execution also for the board targets with the FLPR core HW (for example, nRF54L15 and nRF54H20 SoCs).
+
+.. note::
+   FLPR code is run from RAM.
+
+   This option is not supported for the following board targets that include an SoC with the FLPR core:
+
+     * ``nrf54h20dk/nrf54h20/cpuapp``
```

Comment:
Consider mentioning that nRF54H20 FLPR is not yet integrated in the sample, while L05 and L10 do not support FLPR at the SDK level

Reply:
discussed offline, decided to not explain what is causing the lack of support for the listed board targets.